tests/E2E/ErrorTest.php

```diff
@@ -40,20 +40,23 @@ public function testAccessTokenInvalid()
      */
     public function testInvalidQuery()
     {
-        $client = new Client('b4c0n73n7fu1', 'cfexampleapi');
+        $client = $this->getClient('cfexampleapi');
+
         $query = (new Query())
             ->where('name', 0);
 
         $client->getEntries($query);
     }
 
     /**
+     * @requires Contentful
```

Comment:
Because they require to speak _directly_ to Contentful, and not a proxy of some kind. I can change this to `@requires Contentful no-proxy`, if you think it looks better